src/components/AdminSettings/SIPBridge.vue

```diff
@@ -208,6 +209,21 @@ export default {
 			this.saveCurrentSetup()
 			showSuccess(t('spreed', 'SIP configuration saved!'))
 		},
+
+		async isDialoutSupported() {
+			const servers = loadState('spreed', 'signaling_servers').servers
+			for (let index = 0; index < servers.length; index++) {
+				const response = await getWelcomeMessage(index)
```

Comment:
this can throw (when the version is even older), as per code in `src/components/AdminSettings/SignalingServer.vue`

So should check that here as well and in case it does, also set `dialOutSupported = false`